src/Orc.Controls/Controls/LinkLabel/LinkLabel.cs

```diff
@@ -339,8 +339,32 @@ private static void OpenBrowserBehaviorImpl(object sender, RoutedEventArgs args)
             return;
         }
 
-        var destinationUrl = hyperlinkSender?.NavigateUri ?? linklabelSender?.Url;
-        if (destinationUrl is null || string.IsNullOrEmpty(destinationUrl.ToString()))
+        var uri = linklabelSender?.Url;
+        if (uri is null)
+        {
+            return;
+        }
+
+        if (!uri.IsAbsoluteUri || !(uri.Scheme.StartsWith("http", StringComparison.OrdinalIgnoreCase) 
```

Comment:
uri.Scheme.StartsWith("http", StringComparison.OrdinalIgnoreCase) || uri.Scheme.StartsWith("https", StringComparison.OrdinalIgnoreCase)  === uri.Scheme.StartsWith("http", StringComparison.OrdinalIgnoreCase)

Reply:
I think Catel even has StartsWithIgnoreCase and StartsWithAnyIgnoreCase